lib/reqTemplate.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+
+var URI = require('swagger-router').URI;
+var rbUtil = require('./rbUtil');
+var TAssembly = require('tassembly');
+
+/**
+ * Creates a function that sets a value at the path
+ * or deletes if undefined values is provided.
+ */
+function _setAtPathFun(path) {
+    var ref = path.split('.').map(function(elem) {
```

Comment:
It would be nice to also validate the path to be of the kind of simple form we currently support. Something like `/^(\$\.)?([a-zA-Z][a-zA-Z0-9-]*)(\.[a-zA-Z][a-zA-Z0-9-]*)*$/` ?